### PR TITLE
Sessions Integration - Client

### LINKED ISSUE #49: Sessions Integration - Client
Integrating client-side sessions stuff into the client. This largely entails implementing some additional reading and sending of network stuff.

### PR DESCRIPTION
**Pull request by sinasabet81**
I wasn't able to test this very well, since I kept encountering DB errors whenever standing the application up in Nitrous, which I'm assuming was because of a missing startup script. I implemented this based on the spec Alex and I discussed, let me know if you guys encounter any errors.

Closes #49

### CODE REVIEW

File: src/webClient/index.js [hunk 1]
```diff
@@ -98,7 +107,10 @@ const postQuery = (query) => {
       throw new Error('Error with posting query');
     })
     .then(res => res.json())
-    .then(obj => obj.id)
+    .then(obj => {
+      updateSession(obj.session_id);
```
Comment (sinasabet81) [inline src/webClient/index.js]: Updating the state's session value using the session_id of the initial post's json response

**Comment by CtrlC-Root** [comment]
It's missing the commands to set up the initial modules in the backend's database. You can compare `nitrous-post-create.sh` with `vagrant/web.sh` to see what's missing (look at the bottom).

**Comment by CtrlC-Root** [comment]
Code looks good and works. 👍

**Comment by CtrlC-Root** [comment]
@sinasabet81 what would be neat is if the "Submit" button changed to "Respond" when the user is in a session. maybe you could also add a "Start Over" button that clears the session.

**Pull request merged**